changelog.d/12789.misc

```diff
@@ -0,0 +1 @@
+The hash_password script now returns an exception when it is called without specifying a config file.
```

Comment:
Minor nitpick: "returns an exception" is a Python implementation detail, rather than an externally-visible property of the script.

```suggestion
The `hash_password` script now fails when it is called without specifying a config file.
```